luigi/parameter.py

```diff
@@ -893,6 +893,51 @@ def serialize(self, e):
         return e.name
 
 
+class EnumListParameter(Parameter):
+    """
+    A parameter whose value is a str-separated (default ",") list of :class:`~enum.Enum`.
+
```

Comment:
Maybe add that the values have to come from the same enum.

Reply:
Did you consider this?

Reply:
Will note that